### PR TITLE
Update cronjob with defaults information

### PR DESCRIPTION
**Pull request by soltysh**
This is a followup to https://github.com/kubernetes/kubernetes.github.io/pull/5475 that updates the default values that were merged in https://github.com/kubernetes/kubernetes/pull/52533

@steveperry-53 and @janetkuo ptal, since the two of you reviewed the original PR.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5556)

### CODE REVIEW

File: docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ apply to already started executions. Defaults to false.
 
 ### Jobs History Limits
 
-The `.spec.successfulJobsHistoryLimit` and `.spec.failedJobsHistoryLimit` fields are optional. These fields specify how many completed and failed jobs should be kept.
-
-By default, there are no limits, and all successful and failed jobs are kept. However, jobs can pile up quickly when running a cron job, and setting these fields is recommended. Setting a limit to `0` corresponds to keeping none of the corresponding kind of jobs after they finish.
+The `.spec.successfulJobsHistoryLimit` and `.spec.failedJobsHistoryLimit` fields are optional.
+These fields specify how many completed and failed jobs should be kept.  By default, they are
+set to 3 and 1 accordingly.  Setting a limit to `0` corresponds to keeping none of the corresponding
```
Comment (janetkuo) [inline docs/concepts/workloads/controllers/cron-jobs.md]: s/accordingly/respectively/g ?

**Comment by steveperry-53** [comment]
@soltysh We're ready to merge this after you make the one small change suggested by @janetkuo.

accordingly -> respectively

File: docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -189,6 +189,7 @@ apply to already started executions. Defaults to false.
 
 ### Jobs History Limits
 
-The `.spec.successfulJobsHistoryLimit` and `.spec.failedJobsHistoryLimit` fields are optional. These fields specify how many completed and failed jobs should be kept.
```
Comment (janetkuo) [inline docs/concepts/workloads/controllers/cron-jobs.md]: Is the statement in "Deleting a CronJob" section (L107) still true? 

```
This stops new jobs from being created. However, running jobs won't be stopped, 
and no jobs or their pods will be deleted.
```

IIUC it's now fixed and `kubectl delete cronjob` now deletes its jobs and pods.

This can be a separate PR (and target `master` branch).

**Comment by steveperry-53** [comment]
@janetkuo, Are the tech issues resolved here, or would you like me to keep this open?
As far as Doc issues go, I'm willing to let the s/accordingly/respecively/ issue slide in the interest of getting this merged. Thanks.

@soltysh Last call.

**Comment by janetkuo** [comment]
@steveperry-53 It is fine to merge this PR now, but there are still some tech issues (what I commented above) that need to be fixed.
